topics/scikit-learn/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David @cournape Cournapeau, The scikit-learn developers
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://github.com/scikit-learn/scikit-learn/releases
```

Comment:
this seems to be the correct URL: http://scikit-learn.org/

Reply:
It's http. We shouldn't encourage http without TLS.

Reply:
@KOLANICH Regardless, please change it, thanks.

Reply:
@KOLANICH Still missing this change, thanks.